assets/app/actions/dispatchActions.js

```diff
@@ -0,0 +1,72 @@
+import { dispatch } from '../store';
```

Comment:
I expect to have some more clean up work to do on that project after I extend its functionality a touch, so I wouldn't say it's quite where it will end up. However, it's conceptually something I'm happy to get behind.